django/thunderstore/repository/tasks.py

```diff
@@ -1,12 +1,48 @@
 from celery import shared_task
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from ulid2 import generate_ulid_as_uuid
 
 from thunderstore.repository.api.experimental.tasks import (
     update_api_experimental_caches,
 )
 from thunderstore.repository.api.v1.tasks import update_api_v1_caches
 
+User = get_user_model()
+
 
 @shared_task
 def update_api_caches():
     update_api_v1_caches()
     update_api_experimental_caches()
+
+
+def _create_ghost_user_username(id_: str) -> str:
+    return f"Ghost User {id_}"
+
+
+def _create_ghost_user_email(id_: str) -> str:
+    # The ID used in ghost user usernames and emails are not the same as their `User.id`
+    # This is because `User` still uses an autoincrement ID
+    return f"{id_}.gu@thunderstore.io"
+
+
+@shared_task
+def delete_user(user_id: int) -> None:
```

Comment:
I'm not sure if this is the best place for this task

Reply:
`thunderstore.account.tasks` would seem more fitting to me